posthog/settings/web.py

```diff
@@ -421,3 +421,5 @@ def static_varies_origin(headers, path, url):
 HOG_TRANSFORMATIONS_CUSTOM_HOG_ENABLED = get_from_env(
     "HOG_TRANSFORMATIONS_CUSTOM_HOG_ENABLED", False, type_cast=str_to_bool
 )
+# Teams allowed to modify transformation code (comma-separated list of team IDs)
+HOG_TRANSFORMATIONS_CUSTOM_ENABLED_TEAMS = get_list(os.getenv("HOG_TRANSFORMATIONS_CUSTOM_ENABLED_TEAMS", "2"))
```

Comment:
We try not to hardcode a default like this as it affects local dev (where team 2 isn't your local team). Instead leave it blank and do it as an env var in charts